<?xml version="1.0" encoding="UTF-8"?>
<!--
SPDX-FileCopyrightText: 2020 Ivan Čukić <ivan.cukic@kde.org>

SPDX-License-Identifier: GPL-2.0-only OR GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
-->
<ui version="4.0">
 <class>VaultDeletionWidget</class>
 <widget class="QWidget" name="VaultDeletionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>150</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="text">
      <string>This action &lt;b&gt;cannot&lt;/b&gt; be undone. This will permanently delete the selected vault!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelConfirm">
     <property name="text">
      <string>Please type in the name of the vault to confirm:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="textVaultNameConfirmation"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDeleteVault">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete this vault</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
